content/pt-br/docs/reference/glossary/shuffle-sharding.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Shuffle-sharding
+id: shuffle-sharding
+date: 2020-03-04
+full_link:
+short_description: >
+  Uma técnica para atribuir requisições para filas que proporciona melhor isolamento do que hashing módulo do número de filas.
+
+aka:
+tags:
+- fundamental
+---
+Uma técnica para atribuir requisições para filas que proporciona melhor isolamento do que hashing módulo do número de filas.
```

Comment:
```suggestion
Uma técnica para atribuir requisições para filas que proporciona melhor isolamento do que efetuar a operação módulo (resto da divisão) do _hash_ da requisição pelo número de filas.
```

How would you feel about explaining further what the modulo operation is about?
I feel like writing "módulo" here without an extra explanation might be confusing depending on the context (or lack of) that the reader is carrying. I forgot to add this to the `short_description` field but it would have to be updated there too to match here (minus the formatting bits that might break).